docs/source/configuration/experimental_query_planning_mode.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Experimental Query Planning Mode 
+subtitle: Switch between legacy and native query planning
+noIndex: true
+---
+
+Apollo Router is in the early stages of transitioning to a native query planner,
+replacing the existing, legacy, planner.
+
+As part of the efforts to ensure correctness and stability of the new planner,
+v1.53.0 of the router enables both planners to run in parallel in order
+compare them. The native planner's result is discarded after the comparison and
+only the legacy planner is used to execute requests.
+
+The native planner uses a single thread in the cold path of the router and has a
+bounded queue of 10. If the queue is full, the router simply does not run the
+comparison. This is done to avoid excessive resource consumption.
+
+You may want to turn off this mode if you see a significant spike in CPU
+utilization. This can happen, if, for example, an erroneous operation fails to
+complete planning in the native planner's background thread, increasing this
```

Comment:
```suggestion
You can disable the native query planner by configuring your `router.yaml` to use just `legacy` planning. You may want to disable it to avoid spikes in CPU utilization, for example if an erroneous operation fails to complete planning in the native planner's background thread.
```